mods/doors/init.lua

```diff
@@ -669,12 +671,13 @@ function doors.register_fencegate(name, def)
 		connect_sides = { "left", "right" },
 		groups = def.groups,
 		sounds = def.sounds,
-		on_rightclick = function(pos, clicker)
+		on_rightclick = function(pos, node, clicker, itemstack, pointed_thing)
```

Comment:
As first comment. Here you can see that 'node' does not need to be passed, so 'node' can be removed where-ever it is not used (see first comment code).